From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@alexandair__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

PRmerger commented:
@alexandair : Thanks for your contribution to the Azure documentation! The author, @cynthn, has been notified to review your proposed change.

cynthn commented:
#sign-off Thanks @alexandair for catching that! :)

andretms committed:
Fixing a few bugs in Android and Windows Desktop

andretms committed:
UWP v0.1

andretms committed:
Preview of Full Page UWP v2 Guided Setup

andretms committed:
Fixing hyperlinks

andretms committed:
Merge branch 'master' into andret-UWP

andretms committed:
Updating TOC

andretms committed:
Removing UWP article parts

andretms committed:
Fixing language, grammar

andretms committed:
Fixing content issues

andretms committed:
Fixing Typos

mrbullwinkle committed:
missing word

ShawnJackson committed:
edit pass: pim-resource-roles-batch1

ShawnJackson committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into pim-resource-roles-batch1

ShawnJackson committed:
edit pass: pim-resource-roles-batch1

ShawnJackson committed:
Acrolinx fixes

ShawnJackson committed:
Update pim-resource-roles-approval-workflow.md

ShawnJackson committed:
Update pim-resource-roles-activate-your-roles.md

AjayKumar-N committed:
(ForumsTeam) Broken URL in the "Note" section "KB" patches

In the referenced URL https://docs.microsoft.com/en-us/azure/app-service/app-service-patch-os-runtime#more-resources
Under the Note section > The ' "KB" patches' link is broken and has to be replaced with an appropriate URL.

Current URL : https://technet.microsoft.com/security/bulletins.aspx
Replacement URL: https://github.com/projectkudu/kudu/wiki/Azure-Web-App-sandbox#writing-to-registry
( As it is a platform locked down) 
Or https://docs.microsoft.com/en-us/security-updates/SecurityBulletins/securitybulletins (General doc to Security Bulletins).

AjayKumar-N committed:
(ForumsTeam) Incorrect URL for latest feature and fixes (request for auto-url updation)

In the reference document, https://docs.microsoft.com/en-us/azure/site-recovery/hyper-v-azure-support-matrix#provider-and-agent
Under the "Provider and agent" section, the 'latest features and fixes' points to an older update (21), but it should instead point to (23) update. 

Currently URL points to : https://support.microsoft.com/en-us/help/4051380/update-rollup-21-for-azure-site-recovery
Replacement URL: https://support.microsoft.com/en-us/help/4091311/update-rollup-23-for-azure-site-recovery
The intent is to make this "https://aka.ms/latest_asr_updates" up-to-date.
Thank you!

ktoliver committed:
edit pass: pim-resource-roles-batch3-b

rayne-wiselman committed:
adding first draft of migrate lift and shift scenario

rayne-wiselman committed:
Update migrate-scenarios-lift-and-shift.md

mmacy committed:
[ACR] ACR Build tutorials 1 & 2

mmacy committed:
[ACR] add articles to TOC

danieldobalian committed:
removing appid claim note

billmath committed:
adding updates per o365 geo location

billmath committed:
adding toc entry

andretms committed:
Merge from master

mmacy committed:
[ACR] integrate review comments - tut #1

mmacy committed:
[ACR] minor fixes

lnalepa committed:
Updated integrating applications doc

tdykstra committed:
add method signature explanation

tdykstra committed:
fix link

andretms committed:
Adjusting content

andretms committed:
Merge from master

andretms committed:
Fixing some issues with content

andretms committed:
Updates to content and known issues

mmacy committed:
[ACR] integrate review comments - tutorial 2

mmacy committed:
[ACR] adding tutorial 3

bitsforthought committed:
[Azure Search] Updating .NET SDK migration guide and How-to for version 5

mmacy committed:
[ACR] azurecli-interactive

tdykstra committed:
reorganize and reword method signature sections

tdykstra committed:
rework link text

tdykstra committed:
clarify that parm list not exhaustive

tdykstra committed:
shorten h2

ktoliver committed:
resolve merge conflict

mmacy committed:
[ACR] fix build-task

mmacy committed:
resolve merge conflict (graph-explorer images)

mmacy committed:
[ACR] tut 1: minor fixes

mmacy committed:
[ACR] tut 2 fixes

MicrosoftGuyJFlo committed:
[MFA] Update to TOC

Commit:
Task 906: Split caching rules into tutorial and how to articles

rayne-wiselman committed:
Merge branch 'rayne-migration-scenario-april17' of https://github.com/rayne-wiselman/azure-docs-pr into rayne-migrate-scenarios-april18

mmacy committed:
[ACR] scenario explanation

mmacy committed:
[ACR] grammar

mmacy committed:
[ACR] more steps

mhko committed:
Update docs on synonyms feature for 2017-11-11 GA.

mhko committed:
Use the new api-version in examples.

mmacy committed:
[ACR] tech process completion

mhko committed:
Update search-api-migration

Commit:
Adding support for compression

markgalioto committed:
adding Backup Azure Stack workloads article

mmacy committed:
[ACR] fix typos

MicrosoftGuyJFlo committed:
[MFA] Update to TOC

mattbriggs committed:
adding new row

markgalioto committed:
fixing the broken link in TOC.yml

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into dean-906

jeahyoun committed:
Update context file to final version for pre-PR merge test.

rayne-wiselman committed:
adding comments

Commit:
Update analysis-services-vnet-gateway.md

rayne-wiselman committed:
updating TOC

mmacy committed:
[ACR] add base image build

mmacy committed:
[ACR] fix container image name

rayne-wiselman committed:
Update migrate-scenarios-lift-and-shift.md

Commit:
Task 906: Edits

bitsforthought committed:
[Azure Search] Addressing PR feedback on SDK docs

MicrosoftGuyJFlo committed:
[MFA] Update to TOC

TerryLanfear committed:
updates

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into jit-asc

mimig1 committed:
Cosmos DB: Moving Unique Keys back to concepts, and removing duplication

rayne-wiselman committed:
Update migrate-scenarios-lift-and-shift.md

Commit:
Removed unsupported version from release notes

markgalioto committed:
updates and improved formatting

markgalioto committed:
fixing the code snippet

markgalioto committed:
improving some wording

Commit:
added file to toc

Commit:
added file to toc

markgalioto committed:
last pass for acrolinx updates

rayne-wiselman committed:
Update migrate-scenarios-lift-and-shift.md

rayne-wiselman committed:
Update migrate-scenarios-lift-and-shift.md

rayne-wiselman committed:
adding graphics

rayne-wiselman committed:
updating graphic files

tfosmark committed:
Merge pull request #38509 from brjohnstmsft/5x-sdk-migrate

[Azure Search] Updating .NET SDK migration guide and How-to for versi…

Commit:
quickfix02

ktoliver committed:
Merge pull request #38514 from MicrosoftGuyJFlo/MFATOC

[MFA] Update to TOC

PRMerger12 committed:
Merge pull request #38200 from danieldobalian/idtokenclaims

removing appid claim note

ktoliver committed:
Merge pull request #38578 from markgalioto/apr202018

adding Backup Azure Stack workloads article

twooley committed:
[Azure AD B2B] Minor article update

KumudD committed:
updated content

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into QQSLBPS0420

Commit:
changes for 6.2RTO release

PRMerger15 committed:
Merge pull request #38619 from MarkusVi/quickfix02

quickfix02

Commit:
Update TOC.md

rmca14 committed:
Merge pull request #38586 from Minewiskan/patch-2

Update analysis-services-vnet-gateway.md

tfosmark committed:
Merge pull request #38587 from rayne-wiselman/rayne-migrate-scenarios-april18

Rayne migrate scenarios april18

ktoliver committed:
Merge pull request #38600 from mimig1/04202018toc

Cosmos DB: Moving Unique Keys back to concepts, and removing duplication

PRMerger17 committed:
Merge pull request #38620 from twooley/twb2bupdate

[Azure AD B2B] Minor article update

tfosmark committed:
Merge pull request #38603 from jeffgilb/420

Removed unsupported version from release notes

mhko committed:
Update date stamps in updated docs. Incorporate review suggestions.

rmca14 committed:
Merge pull request #38612 from davidmu1/fileadd

Fileadd

TerryLanfear committed:
fix

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into jit-asc

KumudD committed:
fixed formatting

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into QQSLBPS0420

mhko committed:
Update wordings around user voice per review comment.

mmacy committed:
[ACR] tighten up the graphics on level 3

mhko committed:
Merge branch 'master' into master

rayne-wiselman committed:
removing couple of dup files, fixing typos

SaurabhSharma-MSFT committed:
(ForumsTeam) Modified the hyperlink

Modified the hyperlink against [Comparing generally available features of the Free, Basic, and Premium editions] to redirect to correct Pricing url - https://azure.microsoft.com/en-us/pricing/details/active-directory/

RicksterCDN committed:
fixing cloudinit example doc

Commit:
Merge pull request #38601 from TerryLanfear/jit-asc

Jit asc

HJToland3 committed:
sqldbmi prerequisites fix

tfosmark committed:
Merge pull request #38623 from KumudD/QQSLBPS0420

Create a Load Balancer - PowerShell

tfosmark committed:
Merge pull request #38631 from rayne-wiselman/rayne-typos

removing couple of dup files, fixing typos

tfosmark committed:
Merge pull request #38624 from Minewiskan/patch-3

Update TOC.md

mhko committed:
Incorporate review comment.

mhko committed:
Merge branch 'master' of https://github.com/mhko/azure-docs-pr

christiankuhtz committed:
tweak

ktoliver committed:
Update pim-resource-roles-approval-workflow.md

tynevi committed:
Merge pull request #38588 from jeahyoun/jeahyoun/msi-context-update

Update context file to final version for pre-PR merge test.

KumudD committed:
tweak

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into QSPSLBupdate

ShawnJackson committed:
Merge pull request #37793 from ktoliver/pim-resource-roles-batch3-b

edit pass: pim-resource-roles-batch3

ktoliver committed:
Merge pull request #37593 from ShawnJackson/pim-resource-roles-batch1

edit pass: pim-resource-roles-batch1

Commit:
Merge pull request #38638 from KumudD/QSPSLBupdate

Quickstart - Load Balancer update

PRMerger12 committed:
Merge pull request #38637 from christiankuhtz/patch-117

tweak

PRMerger12 committed:
Merge pull request #38341 from lnalepa/lesia-integrating-applications

Updated integrating applications doc

Commit:
Removing another limitation

PRMerger17 committed:
Merge pull request #38576 from JSeb225/patch-29

Adding support for compression

Commit:
Task 1122: Create billing FAQ article

moderakh committed:
Azure Cosmos DB Java Async SDK 1.0.1 release

PRMerger18 committed:
Merge pull request #38646 from moderakh/patch-13

Azure Cosmos DB Java Async SDK 1.0.1 release

AjayKumar-N committed:
(ForumsTeam) Update on rollupdate KB article

https://support.microsoft.com/help/4091311/update-rollup-23-for-azure-site-recovery

AjayKumar-N committed:
(ForumsTeam) 

The current URL https://technet.microsoft.com/security/bulletins.aspx - lands on the page which mentions "Bulletins released before 2017 are now archived".
Possible replacement URL:  https://docs.microsoft.com/security-updates/SecurityBulletins/securitybulletins (General doc to Security Bulletins).
Or can be replaced with this URL: https://github.com/projectkudu/kudu/wiki/Azure-Web-App-sandbox#writing-to-registry , as it is a App Service platform locked down on registry and this document gives overview on the same.

PRMerger16 committed:
Merge pull request #38585 from mattbriggs/mdb_20180420_storageupdate

adding new row

christiankuhtz committed:
Limitations update

PRMerger19 committed:
Merge pull request #37755 from AjayKumar-N/patch-28

(ForumsTeam) Incorrect URL for latest feature and fixes (request for …

Commit:
Merge pull request #38654 from christiankuhtz/patch-119

Limitations update

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into dean-906

Commit:
Task 906: Edits, fix TOC

Commit:
Task 906: Edits

PRMerger15 committed:
Merge pull request #38632 from SaurabhSharma-MSFT/patch-2

(ForumsTeam) Modified the hyperlink

PRMerger10 committed:
Merge pull request #38622 from v-payc/updatefor6.2

changes for 6.2RTO release

mikeurnun committed:
fixes #7454

fixes MicrosoftDocs/azure-docs#7454

GitHubber17 committed:
Merge pull request #38166 from mmacy/acr-build-preview

[ACR] ACR Build tutorials

GitHubber17 committed:
Merge pull request #38365 from andretms/andret-UWP

Guided Setup for UWP

GitHubber17 committed:
Merge pull request #38634 from HJToland3/master

sqldbmi prerequisites fix

GitHubber17 committed:
Merge pull request #38572 from mhko/master

[Azure Search] Update Azure docs for 2017-11-11 GA

rimman committed:
minor gremlin edits

PRMerger18 committed:
Merge pull request #38666 from rimman/intro_04222018

minor gremlin edits

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into dean-1122

Commit:
Remove en-us from link

Commit:
Merge pull request #38660 from mike-urnun-msft/patch-8

fixes #7454

Sogup committed:
Update backup-azure-arm-vms-prepare.md

Commit:
Merge pull request #38672 from Sogup/patch-67

Update backup-azure-arm-vms-prepare.md

dominicbetts committed:
Fix link

PRMerger17 committed:
Merge pull request #38675 from dominicbetts/fix-issue-7541

Fix issue 7541

PRMerger19 committed:
Merge pull request #37752 from AjayKumar-N/patch-27

(ForumsTeam) Broken URL in the "Note" section "KB" patches

v-geberr committed:
info from Nayer

rjagiewich committed:
Merge pull request #38269 from billmath/geo

adding updates per o365 geo location

mumian committed:
[HDI] incorporate CSS feedback

GitHubber17 committed:
Merge pull request #38645 from dksimpson/dean-1122

Task 1122: Create billing FAQ article

PRMerger13 committed:
Merge pull request #38690 from mumian/0423-edgenode-jeremy

[HDI] incorporate CSS feedback

GitHubber17 committed:
Merge pull request #38656 from dksimpson/dean-906

Split caching rules into tutorial and how to articles

GitHubber17 committed:
Merge pull request #38633 from RicksterCDN/cloudinit-update-fix

fixing cloudinit example doc

Commit:
reportin05

billmath committed:
updating

Commit:
Merge pull request #38356 from tdykstra/patch-36

add method signature explanation

billmath committed:
fixing link

PRMerger10 committed:
Merge pull request #38682 from v-geberr/0423-review-endpoint-utterances

info from Nayer

PRMerger13 committed:
Merge pull request #34906 from mrbullwinkle/mrb_03_07_2018

missing word

PRMerger13 committed:
Merge pull request #38699 from billmath/b2b

updating

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

v-alje committed:
Merge pull request #38703 from MicrosoftDocs/FromPublicRepo

From public repo

rjagiewich committed:
Merge pull request #38696 from MarkusVi/reportin05

reportin05

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo
